### PR TITLE
feat: Added Ardoq blip

### PR DESCRIPTION
**Pull request by l0kal**

### CODE REVIEW

File: radar/qa/ardoq.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Ardoq
+blip:
+  - date: 2019-06-20
+    ring: ASSESS
+description: |
+  Ardoq is a collaborative, easy-to-use and intuitive tool for documenting your projects,
+  business processes, applications, services, infrastructure or anything else that you can document in a structured way.
+rationale: |
+  We have identified a gap within our technical documentation.
+  Detail of dependencies and architecture if low and process diagrams are either lacking or not updated.
```
Comment (Tasseri) [inline radar/qa/ardoq.yaml]: Details*
is low*

File: radar/qa/ardoq.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  We need a tool that assists capturing this information, with minimum effort and maximum value to an audience.
+  As RS is about to go into a huge phase of change on an architecture level
```
Comment (Tasseri) [inline radar/qa/ardoq.yaml]: Not only RS, I would change it to the organisation.

File: radar/qa/ardoq.yaml (hunk 1 above)
Comment (l0kal) [reply radar/qa/ardoq.yaml]: Changed wording.

File: radar/qa/ardoq.yaml (hunk 2 above)
Comment (l0kal) [reply radar/qa/ardoq.yaml]: should be fixed now.

**Pull request merged**